src/kontrol/foundry.py

```diff
@@ -187,7 +187,11 @@ def find_enums(dct: dict) -> None:
             contract_name = json_path.split('/')[-1]
             contract_json = json.loads(Path(json_path).read_text())
             contract_name = contract_name[0:-5] if contract_name.endswith('.json') else contract_name
-            contract = Contract(contract_name, contract_json, foundry=True)
+            try:
+                contract = Contract(contract_name, contract_json, foundry=True)
+            except KeyError:
+                _LOGGER.warning(f'Skipping non-compatible JSON file for contract: {contract_name} at {json_path}.')
```

Comment:
Would it make sense to suggest downgrading Solidity version to 0.8.25?

Reply:
Ok, I can't confirm that it's caused by this specific version; let's merge as is and pinpoint the issue later based on user feedback.